Sources/Sentry/SentryTracer.m

```diff
@@ -263,6 +264,15 @@ - (void)captureTransaction
     if (_hub == nil)
         return;
 
+    @synchronized(_children) {
+        for (id<SentrySpan> span in _children) {
+            if (!span.isFinished) {
+                [span finishWithStatus:kSentrySpanStatusDeadlineExceeded];
+                span.timestamp = self.timestamp;
```

Comment:
`l`: Can we add a comment that we need to do this because the unfinished children should have the same end timestamp as their parent transaction.